static/scss/course_details/header.scss

```diff
@@ -0,0 +1,75 @@
+// sass-lint:disable mixins-before-declarations
+.header-block {
+  background-size: cover;
+  background-origin: 50% 50%;
+}
+
+.header-container {
+  padding-top: 85px;
+  padding-bottom: 85px;
+
+  .program-position {
```

Comment:
After you make this an `h5`, remove `display: block;`

Reply:
Done.